src/stores/mapStore.ts

```diff
@@ -0,0 +1,25 @@
+import { create } from 'zustand'
+
+interface MapStore {
+  lat: number
+  lng: number
+  markers: google.maps.marker.AdvancedMarkerElement[]
+  googleMap: google.maps.Map
```

Comment:
```suggestion
  googleMap: google.maps.Map | null
```
googleMap을 초기화 시에는 null로 설정해서 오류 수정할 수 있을 것 같습니다!